docs/build-parachains.md

```diff
@@ -0,0 +1,174 @@
+---
+id: build-parachains
+title: Parachain Development
```

Comment:
Why didn't you use `git mv`?

Reply:
Never needed it before TBH. It's usually picked up by git automagically. I didn't use this for the `build-build-with-polkadot.md` -> `build-guide.md` in this PR, as an example 🤷🏼 
TIL